<?xml version="1.0"?>
<launch>

	<arg name="tracking_frame" default="pioneer/base_link"/>
	<arg name="object_name" default="pioneer/map"/>
	<arg name="max_vel_factor" default="1.3"/>
	<arg name="ideal_distance" default="2"/>

	<node name="relative_pose_publisher" type="relative_pose_publisher.py" pkg="tracking_navigation">
		<param name="tracking_frame" value="$(arg tracking_frame)"/>
		<param name="object_name" value="$(arg object_name)"/>
	</node>

	<include file="$(find tracking_navigation)/launch/move_base_pioneer.launch"/>
	<include file="$(find obstacle_detector)/launch/nodes.launch"/>

	<node name="vel_refac_with_circle_tracking" type="vel_refac_with_circle_tracking.py" pkg="tracking_navigation" output="screen">
		<param name="max_vel_factor" value="$(arg max_vel_factor)"/>
		<param name="ideal_distance" value="$(arg ideal_distance)"/>
	</node>
</launch>
